app/src/main/java/com/example/timemanagementapp/ui/MainActivity.kt

```diff
@@ -143,7 +148,8 @@ class MainActivity : AppCompatActivity() {
         }
 
         fun viewActivityHistory(view: View){
-            val intent = Intent(this, ViewActivityHistory::class.java)
+            val intent =
+                Intent(this, ViewActivityHistory::class.java)
```

Comment:
for properties with long initializers add a line break after '=' sign for better readability

https://kotlinlang.org/docs/reference/coding-conventions.html

Reply:
This one is not too long, so safe to keep on one line.  I find these breaks to be more important:
1) When you have a line that is longer than 80 characters, or
2) When you are chaining multiple method/function calls together.